manifest.json

```diff
@@ -4,7 +4,7 @@
   "manifest_version": 2,
   "name": "Mozilla Rally",
   "version": "0.10.0",
-  "homepage_url": "https://github.com/mozilla-extensions/pioneer-v2-core-example",
+  "homepage_url": "https://github.com/mozilla-rally/rally-core-addon",
```

Comment:
Do you think this should still be a github link or should we use https://addons.mozilla.org/en-US/firefox/addon/mozilla-rally/ ?

Reply:
Good idea!

Reply:
Well, no, this is forbidden :) 

` Links directing to      manifest.j…                
                          "addons.mo…   "addons.mozilla.org"    son                        
                          zilla.org"    are not allowed to be                              
                          is not        used for homepage  `